website/docs/components/form/radio-card/partials/code/component-api.md

```diff
@@ -50,16 +50,22 @@
     Yields an icon inside the card container. For details about its API check the [`FlightIcon`](/foundations/icons?tab=code) component.
   </C.Property>
   <C.Property @name="<[R].Label>" @type="yielded component">
-    A container that yields its content emphasized inside the card. `...attributes` spreading is supported on this component.
+    A container that yields its content emphasized inside the card. 
+    <br/><br/>
```

Comment:
Not trying to solution here, but I think long-term using forced line-breaks isn't going to be super scalable.

Reply:
Unfortunately, this is a limitation of the Component API component.